app/assets/javascripts/index/directions/fossgis.js

```diff
@@ -1,12 +1,12 @@
 // OSRM car engine
```

Comment:
@mmd-osm fossgis.js is not identical with osrm.js. At the time I originally authored it I was playing with the idea of supporting all in one (needs some special casing because of the slightly different URLS and lack of different profiles), but given that the MB instance is unlikely to come back, I punted.